### PR TITLE
Enable :new_consents

### PR DESCRIPTION
**Pull request by tvararu**
Depends on https://github.com/nhsuk/manage-childrens-vaccinations/pull/763.

Switches to the new `ManageConsents` journey and `health_answers` for handling all nurse-facing consent flows. Updates the tests where necessary.

See commits.

### CODE REVIEW

File: app/controllers/dev_controller.rb [hunk 1]
```diff
@@ -32,6 +32,8 @@ def reset
       )
     end
 
+    Flipper.enable :new_consents
```
Comment (misaka) [inline app/controllers/dev_controller.rb:35]: Will this be removed before merging?

File: tests/nurse_consent_no_response.spec.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ async function when_i_select_a_child_with_no_consent_response() {
 }
 
 async function when_i_click_get_consent() {
-  await p.getByRole("link", { name: "Get consent" }).click();
```
Comment (misaka) [inline tests/nurse_consent_no_response.spec.ts:47]: Why is this change not like the others?

Comment (tvararu) [reply tests/nurse_consent_no_response.spec.ts:47]: This one is actually initially a button and then it's a link. It's a button when it's creating a consent, and a link when it's editing one. One alternative is to create two different `when_i_click_*` methods, which would also work

File: app/controllers/dev_controller.rb (hunk 1 above)
Comment (tvararu) [reply app/controllers/dev_controller.rb:35]: We need to keep it in because tests will fail otherwise, since they've been migrated to the new controller and certain edge cases have been updated. It will be removed in the follow-up PR that removes the feature flag completely.

File: tests/nurse_consent_no_response.spec.ts (hunk 2 above)
Comment (misaka) [reply tests/nurse_consent_no_response.spec.ts:47]: If there's a reason for it then it's probably ok to leave. There's always the chance the test breaks if "Get consent" shows up elsewhere, but we can wait for that calamity to hit before we act on it.

**Review by misaka — APPROVED** [review]
🚀 🚀 🚀

**Review by misaka — APPROVED** [review]
🐬
